### ISSUE TITLE
x/mobile/cmd/gomobile: gomobile init => gomobile: exit status 1

### ISSUE THREAD

**Issue by hbfs** [golang/go#10739]
Running OS X 10.10.3

Cloned tip. Bootstrapped with `go version go1.4.2 darwin/amd64` successfully:
```
go version devel +965d00f Thu May 7 02:56:23 2015 +0000 darwin/amd64
```

When running `gomobile init` =>  `gomobile: exit status 1`

**Comment by hbfs** [comment]
Rewinded to 32d6fbc per https://github.com/golang/go/issues/10598 and `gomobile init` completed successfully.

**Comment by rakyll** [comment]
I was not able to reproduce it against the go tip (965d00f82a5efa1a729db71b813bc6c3fd57bcf2) and mobile tip (a18cedb28dd03f0eb991a448197a008062eca52f). Could you sync to those versions and retry? If it fails again, could you do a `gomobile init -v`?

**Comment by hbfs** [comment]
Hmm.. deleted everything and confirmed starting point of go tip (965d00f) and mobile tip (a18cedb28dd03f0eb991a448197a008062eca52f) and it completed successfully with `gomobile init -v`. If I'm able to recreate will post again.. but leaving it closed for now.

**Comment by weitzj** [comment]
I guess it would be good to mention in the documentation that gomobile init needs `GOROOT_BOOTSTRAP` as well

**Comment by rakyll** [comment]
@weitzj, we document the requirement at gomobile docs, see http://godoc.org/golang.org/x/mobile/cmd/gomobile.

**Comment by weitzj** [comment]
Ah. Then it was a misunderstanding. It looked like for building GO I need `GOROOT_BOOTSTRAP`, but not necessary for `gomobile init`.

**Comment by pftbest** [comment]
gomobile docs are not clear. I already had go1.5 installed, so I skipped 'building go1.5' part, and there is no mention that I need to set `GOROOT_BOOTSTRAP` for `gomobile init` anyway. Error message is not clear either. Luckily I found this post on google.

**Comment by rakyll** [comment]
Building cross compilers requires much work, I am wondering whether we can automate this process by checking out the source code if no GOROOT and/or GOROOT_BOOTSTRAP is set.

/cc @crawshaw @hyangah

**Comment by crawshaw** [comment]
Fixed by golang/mobile@743b9bf.

**Comment by sky7987** [comment]
go install gomobile
